packages/frontend-app-api/src/tree/resolveAppNodeSpecs.ts

```diff
@@ -202,7 +204,7 @@ export function resolveAppNodeSpecs(options: {
         existing.params.disabled = Boolean(overrideParam.disabled);
       }
       order.set(extensionId, existing);
-    } else {
+    } else if (!allowUnknownExtensionConfig) {
```

Comment:
I don't think this is needed at this stage, but did you explore whether it would make sense to surface these somehow?

A potential idea to explore down the line is to collect all of these kind of errors are errors that are either throw an halt app startup, or are collected and can be enumerated somehow